packages/common/src/lib/common/password.ts

```diff
@@ -29,27 +29,28 @@
  *
  */
 
-/* jshint -W097 */
-/* jshint strict: false */
-/* jslint node: true */
-'use strict';
+import * as crypto from 'crypto';
```

Comment:
With `esModuleInterop: true` in the root tsconfig (I think it is in @tsconfigs/node), you can shorten this to 
```suggestion
import crypto from 'crypto';
```